command-expansion-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -227,9 +227,14 @@ export class Command<
       string,
       string | undefined,
     ];
+
+    //use to convert doy to month and day
+    const doyDate = new Date(parseInt(year, 10), 0, parseInt(doyStr, 10));
```

Comment:
I think Date is localized, so this might be incorrect unless we also pass the time zone to the ZonedDateTime constructor

Reply:
Thanks for the suggestion. I am converted the date to a UTC date now.

https://github.com/NASA-AMMOS/aerie/pull/194/files#diff-224d12ae69810ebad6855e95127d0a73a31d2653d5207744b5f8f68370bc0727R233-R242